src/app/forms/changeBackground.directive.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('uvpBuilderWeb.forms')
+    .directive('changeBackgroundButton', changeBackgroundButton);
```

Comment:
it's a good idea to use a prefix for the directives, in this case I think we could use `uvp`